recipes/gts/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "gts" %}
+{% set version = "0.7.6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: http://downloads.sourceforge.net/gts/gts-{{ version }}.tar.gz
+    sha256: 059c3e13e3e3b796d775ec9f96abdce8f2b3b5144df8514eda0cc12e13e8b81e
+    patches:
+      - config.in.win32.patch
+  - path: CMakeLists.txt
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}  # [not win]
+    - {{ compiler('c') }}    # [not win]
+    - vs2017_win-64          # [win]
+    - cmake       # [win]
+    - ninja       # [win]
+    - make        # [not win]
+    - pkg-config  # [not win]
+  host:
+    - glib
+  run:
+    - glib
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libgts.so  # [linux]
+    - test -f ${PREFIX}/lib/libgts-0.7.5.0.1.dylib  # [osx]
```

Comment:
Is the version in the name intended here?

Reply:
No, let me see if the build still works without  the version suffix check.

Reply:
@isuruf Looks I don't need the version check. :)